src/main/java/org/liquidengine/legui/layout/DefaultLayoutManager.java

```diff
@@ -0,0 +1,47 @@
+package org.liquidengine.legui.layout;
+
+import java.util.List;
+import org.liquidengine.legui.component.Component;
+import org.liquidengine.legui.component.Frame;
+import org.liquidengine.legui.component.Layer;
+
+/**
+ * Default layout manager.
+ *
+ * @author Aliaksandr_Shcherbin.
+ */
+public class DefaultLayoutManager extends LayoutManager {
+
+    /**
+     * Used to layout frame layers and all of their child components.
+     *
+     * @param frame frame to lay out.
+     */
+    @Override
+    public void layout(Frame frame) {
+        for (Layer layer : frame.getAllLayers()) {
+            layout(layer.getContainer());
+        }
+    }
+
+    /**
+     * Used to layout component and all of his child components.
+     *
+     * @param component component to lay out.
+     */
+    private void layout(Component component) {
```

Comment:
Why this method is not in LayoutManager? can you only layout Frames? Since the interface Layout allow to modify components the LayoutManager should be able to modify only a subtree of components instead of all the components on a Frame

Reply:
I'll add it to interface.